src/components/PopoverMenu/PopoverMenuItem.vue

```diff
@@ -68,7 +68,8 @@
 		</span>
 
 		<!-- If item.action is set instead, a button will be used -->
-		<button v-else-if="item.action" class="menuitem" @click.stop.prevent="item.action">
+		<button v-else-if="item.action" class="menuitem" :class="{'active':(item.active === true)}"
```

Comment:
you could only check with `:class="{'active': item.active}"` :thinking:

Reply:
changed -- thank you!